### ISSUE TITLE
Transfer focus to the PhpStorm window

### ISSUE THREAD

**Issue by puld** [sanduhrs/phpstorm-url-handler#10]
When the handler is triggered, manually open the window PhpStorm.
It is much more convenient to become more comfortable when you automate this process using _wmctrl_. I locally modified the command and now it looks like this:
```sh
if type phpstorm > /dev/null; then
    /usr/bin/env phpstorm --line "${line}" "${file}" && wmctrl -a PhpStorm
elif type pstorm > /dev/null; then
    /usr/bin/env pstorm --line "${line}" "${file}" && wmctrl -a PhpStorm
fi
```

**Comment by IvanAlekseevichPopov** [comment]
@puld , I came to the same decision, but it is not ideal. If you has many opened phpstorm projects same time - focus can jump any of window scope. This behavior can even interfere, all the while shifting the focus to the wrong window.

**Comment by alexbusu** [comment]
This is awesome. Since it is not ideal, at least it is closer than the current solution (no autofocus at all). Also, when you have multiple PhpStorm projects/windows open, how does the url handler help if it would open the file in _another_ project's window?

**Comment by IvanAlekseevichPopov** [comment]
@alexbusu , In my case it moved focus from right phpstorm window to wrong every time (
Here example - https://www.youtube.com/watch?v=oTEHqzr1hKs&feature=youtu.be

**Comment by sanduhrs** [comment]
Should be fixed with 0f256fe43dfc26064dd2abca2f77b784fcff5576
Thanks for the solution :)
